plugins/tiddlywiki/katex/plugin.info

```diff
@@ -3,5 +3,6 @@
 	"description": "KaTeX: mathematical typography",
 	"author": "JeremyRuston",
 	"core-version": ">=5.0.0",
-	"list": "readme usage"
+	"list": "readme usage",
+	"version": "v0.10.2"
```

Comment:
The version field of the core plugins is set automatically as part of the build process. Perhaps here better to use a new field like "library-version" for the version of the underlying library.